realm/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -4,3 +4,4 @@ distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-bin.zip
+set JAVA_OPTS="-Xmx512m"
```

Comment:
This file only affects gradle wraper(gradlew).

`gradle.properties` affects any gradle execution which is not limited to gradle wrapper.

And I realize that `set FOO` is not a syntax for properties file.
I suspect that this line has no effect at all.